pilot/cmd/pilot-discovery/request.go

```diff
@@ -30,7 +29,6 @@ var (
 		Short: "Makes an HTTP request to Pilot metrics/debug endpoint",
 		Args:  cobra.MinimumNArgs(2),
 		RunE: func(c *cobra.Command, args []string) error {
-			cmd.PrintFlags(c.Flags())
```

Comment:
Why this?

Reply:
istioctl *debug* command (like proxy-config, authn) call `pilot-discovery request` and consume response from stdout. Printing the flag for this command make the response unusable.

Reply:
ic